docs/source/tracking.rst

```diff
@@ -187,19 +187,46 @@ common tasks:
 Running a Tracking Server
 -------------------------
 
-The MLflow tracking server launched via ``mlflow ui`` also hosts REST APIs for tracking runs,
+The MLflow tracking server launched via ``mlflow server`` also hosts REST APIs for tracking runs,
 writing data to the local filesystem. You can specify a tracking server URI
 via the ``MLFLOW_TRACKING_URI`` environment variable and MLflow's tracking APIs will automatically
 communicate with the tracking server at that URI to create/get run information, log metrics, etc.
 
-For example, to launch a run against a local tracking server, launch ``mlflow ui``, set
-``MLFLOW_TRACKING_URI`` to ``http://localhost:5000``, and run:
+When running the remote server, it is recommended that you provide the `--artifact-root` command
+line option. This option will cause clients to log their artifact output (e.g., models) to this
+location which is suitable for large data (such as an S3 bucket or shared NFS file system). If
+you do not provide this option, then clients will write artifacts to `their` local directories,
+which the server can't serve.
+
+An example configuration for a server is as follows:
+
+.. code:: shell
+
+    mlflow server \
+        --host 0.0.0.0 \
+        --artifact-root s3://my-mlflow-bucket/
+
+Note that for the clients and server to access the artifact bucket, you should configure your Cloud
+Provider credentials as normal. For example, S3 can be accessed by setting the ``AWS_ACCESS_KEY_ID``
+and ``AWS_SECRET_ACCESS_KEY`` environment variables, by using an IAM role, or by configuring a default
+profile in `~/.aws/credentials`. See the `AWS docs <https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/setup-credentials.html>`_ for more info.
+
+If running a server in production, we would recommend not exposing the built-in server broadly (as it
+is unauthenticated and unecrypted), and instead putting it behind a reverse proxy like nginx or apache,
+or connecting over VPN. Additionally, you should ensure that the `--file-store` (which defaults to the
+``./mlruns`` directory) points to a persistent (non-ephemeral) disk.
+
+Connecting to a remote server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I think we've capitalized words in our subsection titles elsewhere so do that here if that's the case